.github/workflows/upload-compilation-result.yaml

```diff
@@ -22,7 +22,7 @@ jobs:
       - uses: actions/checkout@v3
       - uses: actions/setup-node@v3
         with:
-          node-version: 16
+          node-version: 18
```

Comment:
Normalement la version 18 de Node nous permet d'utiliser le fetch natif, non ? 

https://dev.to/andrewbaisden/the-nodejs-18-fetch-api-72m

Plus besoin d'isomorphic-fetch ?

Reply:
Effectivement, ma branche de dev était en 16 pendant que la prod est passée en 18.